### PR TITLE
feat - build sample data

### PR DESCRIPTION
**Pull request by wonhyung64**
## What does this PR do?
- build sample pos data

## How to test this? 
- 

## Does anything else need to be done?
-

### CODE REVIEW

File: data/mock_data.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+____	__id	__	__	__	____
```
Comment (RektPunk) [inline data/mock_data.csv:1]: change colnames into english

**Review by RektPunk — CHANGES_REQUESTED** [review]
request changes :)

File: data/mock_data.csv (hunk 1 above)
Comment (wonhyung64) [reply data/mock_data.csv:1]: ok :)

**Review by RektPunk — APPROVED** [review]
awesome

**Pull request merged**